runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogAddressSpaceResponse.java

```diff
@@ -0,0 +1,43 @@
+package org.corfudb.protocols.wireprotocol;
+
+import io.netty.buffer.ByteBuf;
+import lombok.Data;
+
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * Represents the response sent by the Log Unit when the log's address space is requested
+ *
+ * It contains:
+ *     1. A per stream map with each stream's corresponding address space
+ *       (composed of the addresses of this stream and trim mark)
+ *     2. Global Log Tail.
+ */
+@Data
+public class LogAddressSpaceResponse implements ICorfuPayload<LogAddressSpaceResponse>{
+
+    final long logTail;
+    final Map<UUID, StreamAddressSpace> streamsAddressSpace;
```

Comment:
private
P.S. it doesn't matter that Lombok adds `private` modifier, it's always better to make it explicit